exercises/robot-simulator/robot_simulator_test.py

```diff
@@ -2,6 +2,8 @@
 
 from robot_simulator import Robot, NORTH, EAST, SOUTH, WEST
 
+# Tests adapted from `problem-specifications//canonical-data.json` @ v2.0.0
+
```

Comment:
For consistency, can you move the version string down by a line? (ie. so that there are two blank lines before and one after the comment).